app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  has_many :reports
+  has_many :trainee_exams
+  has_many :trainee_subjects
+  has_many :trainee_lessons
+end
```

Comment:
Thêm mối quan hệ nhiều nhiều kiểu https://guides.rubyonrails.org/association_basics.html#the-has-many-through-association ấy c

Reply:
t k nghĩ ra đc nên để tên như nào nên để tạm vậy, c có nghĩ đc tên quan hệ thế nào k